webhooks/ssp_webhook.go

```diff
@@ -53,6 +66,65 @@ func Setup(mgr ctrl.Manager) error {
 		Complete()
 }
 
+// The OLM limits the webhook scope to the namespaces that are defined in the OperatorGroup
```

Comment:
Is that an issue for core KubeVirt too? How do we avoid the creation of multiple KubeVirt CRs when deployed via HCO? Does it use the same approach?

Reply:
It is not. SSP `ValidatingWebhookConfiguration` is created by OLM. The webhook for `Kubevirt` resource is created by `virt-operator` and it only checks deletion, not create or update.